third_party/blink/renderer/farbling/brave_session_cache.cc

```diff
@@ -25,11 +29,15 @@
 #include "third_party/blink/renderer/platform/language.h"
 #include "third_party/blink/renderer/platform/network/network_utils.h"
 #include "third_party/blink/renderer/platform/supplementable.h"
+#include "third_party/blink/renderer/platform/weborigin/security_origin.h"
+#include "third_party/blink/renderer/platform/wtf/casting.h"
 #include "third_party/blink/renderer/platform/wtf/text/string_builder.h"
+#include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
 
 namespace {
 
-const uint64_t zero = 0;
+constexpr uint64_t zero = 0;
+constexpr double maxUInt64AsDouble = UINT64_MAX;
```

Comment:
this should have an explicit cast like https://github.com/brave/brave-core/pull/13737/files#diff-6deb9df40e3d34077b092a41c3d6ebe7e33c532016bb91c37cae2547754b6d8eR248 and also https://github.com/brave/brave-core/pull/13737/files#diff-6deb9df40e3d34077b092a41c3d6ebe7e33c532016bb91c37cae2547754b6d8eR248 should use this

Reply:
Fixed